src/main/rascal/salix/HTML.rsc

```diff
@@ -288,7 +286,7 @@ Attr onSubmit(Msg msg) = event("submit", succeed(msg));
 Attr onBlur(Msg msg) = event("blur", succeed(msg));
 Attr onFocus(Msg msg) = event("focus", succeed(msg));
 
-Attr onCheck(Msg(bool) f) = event("change", targetChecked(f));
+Attr onCheck(Msg(bool) f) = event("check", targetChecked(f));
```

Comment:
@tvdstorm I've fixed a c&p error (the `onCheck` function was defined twice), I hope I generated the right events.